packages/workbox-sw/src/lib/router.js

```diff
@@ -101,6 +102,26 @@ class Router extends SWRoutingRouter {
       if (capture.length === 0) {
         throw ErrorFactory.createError('empty-express-string');
       }
+      // See https://github.com/pillarjs/path-to-regexp#parameters
+      const wildcards = '[*:?+]';
+      const valueToCheck = capture.startsWith('http') ?
+        new URL(capture, location).pathname :
+        capture;
+      const match = valueToCheck.match(new RegExp(`${wildcards}`));
+      if (match) {
+        logHelper.warn({
+          message: `registerRoute() was called with a string containing an ` +
+            `Express-style wildcard character. While this is currently ` +
+            `supported, it will no longer be treated as a wildcard match in ` +
```

Comment:
Simplify the message here.

Suggestion: 
`registerRoute() was called with a string containing an express-style wild card character. In the next version of Workbox express style routes won't be supported by default, instead strings will have to be exact matches. Please switch to regulare expressions`.